website/docs/components/icon/index.md

```diff
@@ -3,7 +3,7 @@ title: Icon
 description: Displays an icon from the Helios icon library to enhance the communication of text and UI elements.
 caption: Used to display an icon.
 links:
-  figma: https://www.figma.com/file/TLnoT5AYQfy3tZ0H68BgOr/Flight-Icons?node-id=164%3A0
+  figma: https://www.figma.com/design/TLnoT5AYQfy3tZ0H68BgOr/HDS-Icons?m=auto&node-id=164-0&t=zAxxcBG32ycFg6Ew-1
```

Comment:
[todo] I have found four instances of `/Flight-Icons?node-id` in our codebase, maybe check if there are other places where this URL should be updated (if you haven't done it yet)

Reply:
I'll take a pass through to make sure these are updated, but FWIW it does redirect to the updated library name and link.

Reply:
Ok, found several more that I've updated